app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/MediaAdapter.kt

```diff
@@ -286,6 +283,32 @@ class MediaAdapter(
         activity.setAs(path)
     }
 
+    private fun resize() {
+        val paths = getSelectedItems().filter { it.isImage() }.map { it.path }
```

Comment:
the isImage() check might not be enough as it includes webm and heic too, which we cannot resize. Not sure about other formats like bmp etc.

Reply:
By webm, you meant webp right?  It's working with the following formats:
```kotlin
val photoExtensions = arrayOf(".jpg", ".png", ".jpeg", ".bmp", ".webp", ".heic", ".heif", ".apng", ".avif")
```

The array above is used by the isImageFast() method which in turn determines the `isImage()` result.

Reply:
ye, webp

Reply:
But heic and avif aren't loaded at all on emulators i.e. x86. On real physical devices, they are visible and can be resized.